include/mbedtls/debug.h

```diff
@@ -165,9 +168,11 @@ void mbedtls_debug_set_threshold( int threshold );
  * \attention       This function is intended for INTERNAL usage within the
  *                  library only.
  */
-void mbedtls_debug_print_msg( const mbedtls_ssl_context *ssl, int level,
-                              const char *file, int line,
-                              const char *format, ... ) MBEDTLS_PRINTF_ATTRIBUTE(5, 6);
+void mbedtls_debug_print_msg(const mbedtls_ssl_context *ssl, int level,
+                             const char *file, int line,
+                             const char *format, ...) MBEDTLS_PRINTF_ATTRIBUTE(
+    5,
```

Comment:
I think that there's a slight problem with macros here.

Reply:
Uncrustify doesn't know that we'd prefer to break the line before `MBEDTLS_PRINTF_ATTRIBUTE`, rather than inside the parentheses. I think that's a reasonable choice in an automated tool. We can add a line break manually.